### PR TITLE
Mention new Maven version requirements

### PR DESCRIPTION
**Pull request by ignasi35**
related to https://github.com/lagom/lagom/pull/1778

### CODE REVIEW

File: src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ If you do not have the correct JDK, download it from the  [Oracle website](http:
 
 
 ## Maven
-Lagom requires Maven 3.2.1 or higher, we recommend that you use at least Maven 3.3. In addition to verifying the version on the command line, check that your IDE is using the correct version.
+Lagom requires Maven 3.5.4 or higher, we recommend that you use at least Maven 3.6. In addition to verifying the version on the command line, check that your IDE is using the correct version.
```
Comment (ignasi35) [inline src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md]:
```suggestion
Lagom requires Maven 3.6.0 or higher. In addition to verifying the version on the command line, check that your IDE is using the correct version.
```

**Comment by dwijnand** [comment]
I assume this is blocked on https://github.com/lagom/lagom/pull/1778 merging first?

**Comment by ignasi35** [comment]
> I assume this is blocked on [lagom/lagom#1778](https://github.com/lagom/lagom/pull/1778) merging first?

that is correct. I forgot to add the reference.

**Comment by ignasi35** [comment]
lagom/lagom#1778 is merged but Lagom `1.5.0` is still not released. This is almost the first page users see so I'd rather hold this merge until `1.5.0` is baked.

**Comment by dwijnand** [comment]
Added to https://github.com/playframework/play-meta/issues/6, then.

File: src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ If you do not have the correct JDK, download it from the  [Oracle website](http:
 
 
 ## Maven
-Lagom requires Maven 3.2.1 or higher, we recommend that you use at least Maven 3.3. In addition to verifying the version on the command line, check that your IDE is using the correct version.
+Lagom requires Maven 3.6.0 or higher. In addition to verifying the version on the command line, check that your IDE is using the correct version.
```
Comment (TimMoore) [inline src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md]: Shouldn't this be changed in lagom/lagom? It will be overwritten in the next build.

Comment (TimMoore) [reply src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md]: I think the page you want to change in this repo is this one: https://github.com/lagom/lagom.github.io/blob/template/src/main/twirl/com/lightbend/lagom/docs/getstartedjavamaven.scala.html

Comment (ignasi35) [reply src/docs/1.4.x/java/resources/gettingstarted/JavaPrereqs.md]: 🤦🏼‍♂️

**Comment by ignasi35** [comment]
I did a close/reopen to test the setup of the CLA hook

**Comment by ignasi35** [comment]
I cherrypicked this change into #179. Closing.

**Pull request closed without merging**